src/components/diagrams/diagram-pane.js

```diff
@@ -460,14 +783,26 @@ export function DiagramPane({
 
                     const substationsIds =
                         studyUpdatedForce.eventData.headers['substationsIds'];
-                    viewsRef.current.forEach((v) => {
-                        const vl = network.getVoltageLevel(v.id);
-                        if (vl && substationsIds.includes(vl.substationId)) {
-                            updateDiagram(vl.id);
-                        }
-                    });
+                    let diagramIds = [];
+                    viewsRef.current
+                        .filter(
+                            (diagramView) =>
+                                diagramView.svgType !== DiagramType.SUBSTATION
```

Comment:
I don't get why we only update SUBSTATIONs now and before we also updated VOLTAGE_LEVELs ?

Reply:
We discussed about